conduit/plugins/importers/algod/algod_importer_test.go

```diff
@@ -752,21 +758,27 @@ func TestGetBlockErrors(t *testing.T) {
 			_, err = testImporter.GetBlock(tc.rnd)
 			noError := assert.ErrorContains(t, err, tc.err)
 
-			// Make sure each of the expected log messages are present
+			// Make sure each of the expected log messages are present in the hookEntries
+			hookEntries := hook.AllEntries()
 			for _, log := range tc.logs {
 				found := false
-				for _, entry := range hook.AllEntries() {
-					fmt.Println(strings.Contains(entry.Message, log))
-					found = found || strings.Contains(entry.Message, log)
+				fmt.Println("~~~\nCurrent log: ", log)
+				for _, entry := range hookEntries {
+					logIsSubstring := strings.Contains(entry.Message, log)
+					found = found || logIsSubstring
+					fmt.Printf("logIsSubstring=%t, found=%t:\n\t%s\n", logIsSubstring, found, entry.Message)
+				}
+				noError = noError && assert.True(t, found, "(%s) Expected log was not found: '%s'", tc.name, log)
+				if !noError {
+					fmt.Printf(">>>>>WE HAVE A PROBLEM<<<<<<\n")
```

Comment:
Make it a little easier to find the failing needle in the printouts haystack since we aren't using `require` assertions.

Reply:
I think you should use `!found` as the condition, in tests that provide multiple log entries would print `WE HAVE A PROBLEM` for all checks after the first one.

Reply:
b9fd3993a94e282ee2d4a99a81a95301b450767e